seismostats/analysis/estimate_mc.py

```diff
@@ -6,50 +6,42 @@
 import pandas as pd
 
 from seismostats.analysis.estimate_beta import estimate_b_tinti
-from seismostats.utils.binning import normal_round, get_fmd
-from seismostats.utils.simulate_distributions import simulate_magnitudes
+from seismostats.utils.binning import get_fmd
+from seismostats.utils.simulate_distributions import (
+    simulated_magnitudes_binned,
+)
 
 
-def fitted_cdf_discrete(
+def cdf_discrete_GR(
     sample: np.ndarray,
     mc: float,
     delta_m: float,
-    x_max: float | None = None,
     beta: float | None = None,
 ) -> tuple[np.ndarray, np.ndarray]:
     """
-    Calculate the fitted cumulative distribution function (CDF)
-    for a discrete Gutenberg-Richter distribution.
+    Calculate the cumulative distribution function (CDF)
+    for a discrete Gutenberg-Richter distribution at the points of the sample.
 
     Parameters:
         sample:     Magnitude sample
         mc:         Completeness magnitude
         delta_m:    Magnitude bins
-        x_max:      Maximum magnitude, by default None
         beta:       Beta parameter for the Gutenberg-Richter distribution, by
                     default None
 
     Returns:
-        x: x-values of the fitted CDF
-        y: y-values of the fitted CDF
+        x: unique x-values of the sample
+        y: corresponding y-values of the CDF of the GR distribution
     """
 
     if beta is None:
         beta = estimate_b_tinti(
```

Comment:
if beta is estimated, this means that x only contains values that are in the sample